handbook/engineering/releases/index.md

```diff
@@ -127,3 +127,8 @@ This is because:
 - We haven't built the automated testing and update infrastructure to make continuous customer releases reliable.
 
 In the future, we may introduce continuous releases if these issues become surmountable.
+
+### Weekend releases
+
+If the 20th falls on a non-working day, the release captain has the right but not the obligation to change the day (+/- 3 days) that the release images are published.
```

Comment:
Why +/- 3 and not +/- 2? and/or why not the previous Friday or following Monday? Just curious to hear understand the thought process  :) (I do not feel strongly about this and support as written).

Reply:
My thought here was if the release was on a Monday holiday after the weekend we could release on the Thursday / Friday before.